crates/holochain/src/conductor/api/api_external/app_interface.rs

```diff
@@ -211,3 +222,28 @@ pub struct AppAuthentication {
     /// must be provided by Holochain and not the client.
     pub installed_app_id: Option<InstalledAppId>,
 }
+
+/// Combination of zome call parameters with individual fields and the signed form which includes
+/// the serialized bytes of the parameters and the provenance's signature of the serialized bytes.
+#[derive(Clone, Debug)]
+pub struct ZomeCall {
```

Comment:
Moved here from `holochain_conductor_api`. This is a new type that is used internally.